src/athena.hpp

```diff
@@ -96,7 +96,7 @@ struct RegionSize { // aggregate and POD type; do NOT reorder member declaration
 
 // needed for arrays dimensioned over grid directions
 // enumerator type only used in Mesh::EnrollUserMeshGenerator()
-enum CoordinateDirection { X1DIR = 0, X2DIR = 1, X3DIR = 2 };
+enum CoordinateDirection { X0DIR = 0, X1DIR = 1, X2DIR = 2, X3DIR = 3 };
```

Comment:
Consider adding a comment here that X0DIR is ignored, X1DIR is x/r, X2DIR is y/theta and X3DIR is z/phi to prevent future developers from banging their head against this.

Reply:
Better yet, add a comment that `X0DIR` is a placeholder for time in spacetime calculations. :P

Reply:
Then we should call it X0TIME

Reply:
done

Reply:
Time is a direction. The 0 direction. That's standard convention in general relativity. `X0DIR` is the relativist's way.